var/spack/repos/builtin/packages/ncurses/package.py

```diff
@@ -42,6 +42,9 @@ class Ncurses(Package):
 
     patch('patch_gcc_5.txt', when='%gcc@5.0:')
 
+    variant('symlinks', default=False,
+        description='Enables symlinks. Needed on AFS filesystem.')
```

Comment:
This line needs to be indented by 4 more spaces in order to pass the flake8 tests.